airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/util/ConnectorExceptionUtil.kt

```diff
@@ -149,7 +149,8 @@ object ConnectorExceptionUtil {
 
     private fun isTransientSQLException(e: Throwable?): Boolean {
         return (e is SQLException) &&
-            e.message!!.lowercase().contains("An I/O error occurred while sending to the backend")
+            (e.message!!.lowercase().contains("An I/O error occurred while sending to the backend") ||
+                e.message!!.lowercase().contains("temporary file size exceeds temp_file_limit"))
```

Comment:
It would be great to start building up a COST or array of these matchers somewhere

Reply:
Re: [The connector usability layer](https://docs.google.com/document/d/1jFpysRq0uGzgwhjnK3HNk5UmIWMGCevyemoaSkTJZtA/edit)

Reply:
> It would be great to start building up a constant or array of these matchers somewhere

Agreed.